core/src/main/java/io/lionweb/serialization/data/SerializedPropertyValue.java

```diff
@@ -4,34 +4,29 @@
 import java.util.Map;
 import java.util.Objects;
 
-/** This represents the serialization of the value of a property in a Node. */
+/**
+ * This represents the serialization of the value of a property in a Node. This class is immutable
+ * and multiple coinciding value share the same identity, but this is not guaranteed for all
+ * instances.
+ */
 public class SerializedPropertyValue {
```

Comment:
Maybe add a remark to the implementation of `hashCode()` why you can't use `System.identityHashCode()` here?